distributed/worker.py

```diff
@@ -2574,7 +2574,7 @@ async def handle_missing_dep(self, *deps, **kwargs):
             who_has = {k: v for k, v in who_has.items() if v}
             self.update_who_has(who_has)
             still_missing = set()
-            for dep in deps:
+            for dep in list(deps):
```

Comment:
I'm wondering what prompted this `list(...)` here. I could be missing something, but I don't see where the `deps` set gets mutated inside this `for`-loop

Reply:
I went ahead and removed this, we can always add it back if needed